charon/acheron_sql.py

```diff
@@ -233,9 +233,13 @@ def generate_samples_docs(self):
             doc['status'] = 'FRESH'
             doc['analysis_status'] = 'TO_ANALYZE'
 
-            remote_sample=self.get_charon_sample(sample.name)
-            if remote_sample and remote_sample.get('status') == 'STALE' and self.seqruns_for_sample(sample.name) == self.remote_seqruns_for_sample(sample.name):
-                doc['status'] = 'STALE'
+            # doc will not be updated if there is a connection error. Script will continue with the next sample
+            try:
+                remote_sample=self.get_charon_sample(sample.name)
```

Comment:
get_charon_sample() returns None when the request fails, so would this Exception be triggered?

Reply:
OK, thanks for the comments! I will change it